src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -413,6 +545,12 @@ private bool IsFileSource()
 
                 var fullPath = Path.Combine(root, packageFileName);
                 File.Copy(pathToPackage, fullPath, overwrite: true);
+
+                //Indicate that SkipDuplicate is currently not supported in this scenario.
+                if (skipDuplicate)
```

Comment:
You are going to log N warnings, where N is the number of packages. 

That's noisy.

Reply:
Already asked others about this before I committed. The thought was that being noisy is fine since someone is using an option that doesn't make sense.

Reply:
I disagree. 

This is completely unnecessary, 

Let's discuss offline.